api/acl.go

```diff
@@ -72,9 +72,31 @@ func (c *Client) ACLTokens() *ACLTokens {
 	return &ACLTokens{client: c}
 }
 
-// Bootstrap is used to get the initial bootstrap token
-func (a *ACLTokens) Bootstrap(q *WriteOptions) (*ACLToken, *WriteMeta, error) {
+// TODO
+// These values will be redacted when reported in the
+// API or in Nomad's logs
```

Comment:
We don't log request bodies or headers, even in the audit log. The token _isn't_ redacted in the API for responses where the user needs to have the token already. Ex. [Read Self Token](https://www.nomadproject.io/api-docs/acl-tokens#read-self-token). So nothing to do here.

```suggestion
```